cloudinit/sources/DataSourceOpenStack.py

```diff
@@ -280,13 +258,6 @@ def detect_openstack(self, accept_oracle=False):
             return True
         elif util.get_proc_env(1).get("product_name") == DMI_PRODUCT_NOVA:
```

Comment:
I have to make a mental note of this:

is this another way of reading kernel command line parameters?